wallet-new/integration/AddressSpecs.hs

```diff
@@ -8,41 +8,49 @@ import           Universum
 import           Cardano.Wallet.Client.Http
 import           Control.Lens
 import           Test.Hspec
+import           Test.Hspec.QuickCheck (prop)
+import           Test.QuickCheck (withMaxSuccess)
+import           Test.QuickCheck.Monadic (monadicIO, run)
+
 
 import           Util
 
 
 addressSpecs :: WalletRef -> WalletClient IO -> Spec
 addressSpecs wRef wc = do
     describe "Addresses" $ do
-        it "Creating an address makes it available" $ do
+
+        prop "Creating an address makes it available" $ withMaxSuccess 1 $
```

Comment:
Seems like we should define a helper for this:

```haskell
randomTest msg = prop msg . withMaxSuccess 1 . monadicIO
```

Reply:
DONE